orbit-db-cli/.github/workflows/onRelease.yml

```diff
@@ -0,0 +1,19 @@
+name: publish
```

Comment:
may be useful but related to @paulogarithm 's work, is it needed?

Reply:
same